lib/charms/istio_beacon_k8s/v0/service_mesh.py

```diff
@@ -6,44 +6,97 @@
 The service mesh library is used to facilitate adding your charmed application to a service mesh.
 The library leverages the `service_mesh` and `cross_model_mesh` interfaces.
 
-To add service mesh support to your charm, instantiate a ServiceMeshConsumer object in the
+##Consumer
+
+To add service mesh support to your charm, you must add 3 relations in your charmcraft.yaml.
```

Comment:
atm this docstring feels really clear to someone who knows what Istio does and its terminology (mainly, authorization policy), but feels confusing to a newcomer.  What about some more intro material up front?

```suggestion
Service meshes provide a range of capabilities for routing, controlling, and monitoring traffic.  A key feature of many service meshes is the ability to restrict traffic between Pods based on L4 and L7 criteria.  For example, defining that a Pod MetricsScraper can `GET` from Pod MetricsProducer at `/metrics` on port `9090`, but SomeOtherPod cannot.  

The ServiceMeshConsumer object is used to subscribe a charm and its workloads to a related service mesh.  Since it is common for a relation between applications to indicate traffic flow (ex: if DbConsumer Requires a DbProducer), the ServiceMeshConsumer provides an optional way to automate creation of traffic rules based on app relations. 

To add service mesh support to your charm, you must add 3 relations in your charmcraft.yaml.
```

Reply:
We can park this for now and address it in #4 if you'd like, just making a note of it now